file-formats/smbc/type/zif_aff_smbc_v1.intf.abap

```diff
@@ -46,11 +46,14 @@ INTERFACE zif_aff_smbc_v1
       "! Item is created in subpage
       new_page     TYPE string VALUE 'NewPage',
       "! <p class="shorttext synchronized">Inline</p>
-      "! By clicking on 'Create' a new line is created but automatic navigation to subpage is not triggered.
+      "! By clicking on 'Create' a new line is created but automatic navigation to subpage is not performed.
       inline       TYPE string VALUE 'Inline',
       "! <p class="shorttext synchronized">Creation Row</p>
-      "! By clicking on 'Add row' a new line is created but automatic navigation to subpage is not triggered.
+      "! By clicking on 'Add row' a new line is created but automatic navigation to subpage is not performed.
       creation_row TYPE string VALUE 'CreationRow',
+      "! <p class="shorttext synchronized">Empty Rows</p>
+      "! In create or edit mode, two new empty rows are added to the table.
+      inline_creation_rows TYPE string VALUE 'InlineCreationRows',
```

Comment:
Any reason why the short text "Empty Rows" an the constant name and value (inline creation rows) differ in their naming? Can we align this?

Reply:
The corresponding UI5 functionality is described here (section Odata V4): https://sapui5.hana.ondemand.com/sdk/#/topic/cfb04f0c58e7409992feb4c91aa9410b.html
![empty_row](https://github.com/SAP/abap-file-formats/assets/105780276/ceb290b7-7fae-4667-ad4a-89e8b1041ce9)

Reply:
OK, I see. :/